File: C/section08/S08E05.c
#include <stdio.h>

int main() { 
    int vector[10], numbersAndIndexes[10][2], aboveFiftyCounter = 0;

    for (int count = 0; count < 10; count++) {
        printf("Please enter a number for the position %d/10: ", (count+1));
        scanf("%d", &vector[count]);

        if (vector[count] > 50) {
            numbersAndIndexes[aboveFiftyCounter][0] = vector[count];
            numbersAndIndexes[aboveFiftyCounter][1] = count;
            aboveFiftyCounter++;
        }
    }

    if (aboveFiftyCounter > 0) {
        for (int count = 0; count < aboveFiftyCounter; count++) {
            printf("\nThe number %d in the index %d is above 50.", numbersAndIndexes[count][0], (numbersAndIndexes[count][1]));
        }
    } else {
        printf("\nNo number above 50 was entered.");
    }

    return 0;
}

File: Python/section08/S08e05.py
vector = []
numberBiggerThanFifty = []
isNumberBiggerThanFifty = False

for index in range(0,10):
    vector.append(int(input("Please enter a number {0}/10: ".format(index+1))))
    if vector[index] > 50:
        numberBiggerThanFifty.append(vector[index])
        isNumberBiggerThanFifty = True

for index in numberBiggerThanFifty:
        print("Number {0} at index {1} is bigger than 50.".format(index, numberBiggerThanFifty.index(index)))

if isNumberBiggerThanFifty == False:
    print("\nThere is no number bigger than 50.")